client/src/modules/reports/generate/debtorBalanceReport/debtorBalanceReport.js

```diff
@@ -0,0 +1,60 @@
+angular.module('bhima.controllers')
+  .controller('debtorBalanceReportController', DebtorBalanceReportController);
+
+DebtorBalanceReportController.$inject = [
+  '$state', '$sce', 'NotifyService', 'BaseReportService', 'AppCache',
+  'BaseReportService', 'reportData',
+];
+
```

Comment:
@jniles, your code looks correct but there are less comments, comments are very important for helping others to easily understand you do.